test/gtest/uct/ib/test_ib.cc

```diff
@@ -181,12 +166,38 @@ void test_uct_ib::test_address_pack(uint64_t subnet_prefix) {
     free(ib_addr);
 }
 
+void test_uct_ib::test_uct_ib_init() {
```

Comment:
why not use the regular test init/cleanup functions which are already called before/after the test ?

Reply:
> why not use the regular test init/cleanup functions which are already called before/after the test ?

we have to check whether gid_index/pkey/pkey provided to the test is supported on a current setup
So, if we do UCT entities initialization inside regular `init()`, it results as a failure, because, for instance, gid_index=1 isn't supported (but we should just skip such tests) - as it was on `boo31`.